packages/idyll-components/src/index.js

```diff
@@ -15,6 +15,7 @@ export { default as Fixed } from './fixed';
 export { default as Float } from './float';
 export { default as Gist } from './gist';
 export { default as Header } from './header';
+export { default as Heading1 } from './heading1';
```

Comment:
We should export as `H1`, `H2`, ..., `H6`.

Reply:
I was thinking of including all the components i:e H1-H6 in a single file, and exporting from that same file or should I create a different file for each component? what do you think?

Reply:
For compatibility sake I think it would be better if they are in separate files. To avoid repetition you could create a utility function that is used by each of the files to create the exported component. @amitbadala